app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+import { Clock } from "neetoicons";
+import { Typography, Tag, Tooltip, Avatar } from "neetoui";
+
+import { USER_IMG_URL } from "./constants";
+import DropdownMenu from "./DropdownMenu";
+
+const Note = ({ note, handleNoteDeletion }) => {
+  const noteLastCreatedAt = "Created 2 hours ago";
+  return (
+    <div className="m-3 w-full space-y-2 border p-4 shadow-md">
+      <div className="flex justify-between">
+        <Typography style="h3">{note.title}</Typography>
+        <DropdownMenu handleNoteDeletion={handleNoteDeletion} note={note} />
+      </div>
+      <div className="text-gray-500">
+        <Typography style="body2">{note.description}</Typography>
+      </div>
+      <hr />
+      <div className="flex w-full justify-between">
+        <Tag className="bg-gray-100" label="Getting Started" />
+        <div className="flex justify-between pt-1">
+          <Clock className="pt-1" size={15} />
+          <Tooltip
+            content="Wednesday, 10:30AM"
+            followCursor="horizontal"
+            position="bottom"
+          >
+            <Typography className="pr-2 text-gray-500" style="body3">
+              {noteLastCreatedAt}
```

Comment:
Content of tooltip and text displayed in UI doesn't match.

Reply:
Fixed this.